### ISSUE TITLE
BVH to VMD

### ISSUE THREAD

**Issue by hboxgames** [alloystorm/dvvr#168]
Hi !

Thanks for all the great features again !

Let me show you a crazy tool I just found the other day : https://we.tl/t-2HNeCRN0n2

So basically, there's the program, + a test convert I did with a mocap BVH file.
The convert does work, surprisingly, but the motion is very odd for sure. I'm guessing it might be because of the Bone map in the csv file that has to be adjusted.

I wondered if you would be able to figure out what's wrong, and improve this thing, which would give sooo much more motions for all of us !!

Please do have a look
Thanks !!

**Comment by alloystorm** [comment]
Interesting find!
Definitely will check it out. It would be awesome if we can use motion
other than VMD.

**Comment by alloystorm** [comment]
Had a quick look at the BVH format, it seems pretty straight forward. It would be easier to just support this motion format instead of trying to support the convertered VMD. Will let you know if any progress we make.

**Comment by hboxgames** [comment]
Hi !
That's an amazing news !
I'm pretty curious about how we can correctly convert vmd, but you're right if you can make it support directly it's even better

**Comment by alloystorm** [comment]
Where do you download BVH resources? 

I did find some but it would be great to have a variaty of different motion to test with.

**Comment by hboxgames** [comment]
Hi !
I remember not where I get the motion I just sent you before, but here is a page with a bunch of files : 
https://www.deviantart.com/hogarth-mmd/journal/2548-free-downloadable-BVH-files-631005592

I also have some more files so I'm sending them to you for the science ! https://we.tl/t-AexSPZZlw5

**Comment by alloystorm** [comment]
Thanks a lot!

**Comment by hboxgames** [comment]
No worries !
Can't wait for this to be working !!

**Comment by hboxgames** [comment]
Ragdoll feature seems so funny tho ! Any luck with the BVH for next update ??

**Comment by alloystorm** [comment]
Still working on that, doesn't look like it's going to be available next month. 

So far we only support A pose as standard pose for motion, to support BVH we'll have to at least support T pose as well. That can take some time.

**Comment by hboxgames** [comment]
Oh I see !
It still is simpler than making just a working profile in the converter right ? I'm so curious how it can come out.
Also was wondering, is there any facials thzt could come out of it ?

**Comment by alloystorm** [comment]
Made some progress: https://youtu.be/27UIiAibUyk

Is your BVH motions mostly come from Virt-A-Mate content?
I found this creator publishing BVH motions: https://www.patreon.com/noheadnoleg/posts, will be mostly testing with their motions. If you have other recommendations feel free to let me know.

**Comment by hboxgames** [comment]
Yes they do ! 
Oh damn, it's already working pretty much like charm as I can see ! Amazing stuff

**Comment by hboxgames** [comment]
Hi !

Looks like this guy makes some nice NSFW mocaps : https://www.patreon.com/bePatron?c=5526357&fc=true

But I'm not really sure if he's releasing the mocaps in BVH ...

**Comment by hboxgames** [comment]
@alloystorm 
So... I tried out the BVH player. It WORKS LIKE A CHARM. such a great job tho !

By the way, I tried out several VAR files, being awed by the fact that you made them compatible .... Some did work, and some didn't.
I extracted those that weren't working, and they didn't contain any BVH inside although it has motion info (since its supposed to be a dance) So I was wondering what the deal was, they basically have something in "Json" format inside, but no BVH.

Is there something we can do to extract BVH out of those ?
